src/h3/_cy/h3lib.pxd

```diff
@@ -10,7 +9,7 @@ cdef extern from "h3api.h":
     cdef int H3_VERSION_MINOR
     cdef int H3_VERSION_PATCH
 
-    ctypedef uint64_t H3Index
+    ctypedef uint64_t H3int "H3Index"
```

Comment:
So that the Cython code only has one name for this concept. I'm using `H3int` here since the Python library deals with both integer and string representations of an `H3Index`, so and the names `H3int` and `H3str` help make it clear which one it is.